Sources/CALayer+ContentsGravity.swift

```diff
@@ -1,17 +1,14 @@
-//
-//  CALayer+ContentsGravity.swift
-//  UIKit
-//
-//  Created by Geordie Jay on 18.02.18.
-//  Copyright © 2018 flowkey. All rights reserved.
-//
-
-
-extension CALayer {
-    internal enum ContentsGravity: String {
-        case left, center, right, top, bottom, topLeft, topRight
-        case resize, resizeAspectFill, resizeAspectFit = "resizeAspect"
-    }
+/*
+ * Apples CALayerContentsGravity implementation is based on a struct
+ * with a raw representable because of backwards compatibility
+ * We implemented it with an enum, which can be used the
+ * same way as Apples CALayerContentsGravity
+*/
+public enum CALayerContentsGravity {
```

Comment:
This should be `public enum CALayerContentsGravity: String {` to allow initing via `CALayerContentsGravity(rawValue:)` which is still possible in the new Xcode version (via its `RawRepresentable` conformance) -> see note about the `.center` fallback